### PR TITLE
Implemented crud for admin

### PR DESCRIPTION
**Pull request by upendra2610**
Created crud for admin by using remote db.

### CODE REVIEW

File: admin-management-service/src/main/java/com/scaler/adminmanagementservice/controllers/AdminController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import com.scaler.adminmanagementservice.exceptions.NotFoundException;
+import com.scaler.adminmanagementservice.services.AdminService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/admins")
```
Comment (ajeetcloud) [inline admin-management-service/src/main/java/com/scaler/adminmanagementservice/controllers/AdminController.java]: remove from here, its there in admin operations

File: admin-management-service/src/main/java/com/scaler/adminmanagementservice/services/AdminServiceImplementation.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        if (user.isPresent()) {
+            return convertToGenericAdminDto(user.get());
+        }
+        throw new NotFoundException("Admin with id:" + id + " " + "doesn't exist");
+    }
+
+    @Override
+    public List<GenericAdminDto> getAllAdmins() {
+        List<GenericAdminDto> genericAdminDtos = new ArrayList<>();
+        List<User> users = userRepository.findAll();
```
Comment (ajeetcloud) [inline admin-management-service/src/main/java/com/scaler/adminmanagementservice/services/AdminServiceImplementation.java]: findAll() will give you Admin and normal user both. We need only admin.

**Review by ajeetcloud — CHANGES_REQUESTED** [review]
pls do the changes

**Comment by upendra2610** [comment]
I have removed the @RequestMapping and modified the findAll all the users who are admins, Please review once

**Review by ajeetcloud — APPROVED** [review]
LGTM

**Pull request merged**